docs/advisors/checks/multiple-mongod-running-in-a-node.md

```diff
@@ -0,0 +1,31 @@
+# MongoDB - Multiple mongod service running in a single node
+
+## Description
+This check returns a notice if multiple mongod services are running in a single node.
+
+## Resolution
+
+Running multiple mongod services on a single instance can be useful in certain scenarios, such as when you need to isolate different workloads or databases on the same hardware. However, it's important to consider the potential downsides before deciding to run multiple mongod services on a single instance.
+
+Here are some of the potential downsides of running multiple mongod services on a single instance:
+
+- **Resource contention:** Running multiple mongod services on the same hardware can result in resource contention, such as CPU usage, memory usage, and disk I/O. This can impact the performance and stability of all mongod instances on the same instance.
+  - One important point is to configure the wiredTiger Cache Size, since by default each mongod service takes ~50% of the RAM. If other applications and services running on the server might not have enough memory to perform.
```

Comment:
```suggestion
  - One important point is to configure the wiredTiger Cache Size, since by default each mongod service takes ~50% of the RAM. Thus, other applications and services running on the server might not have enough memory to perform.
```